pytket/pytket/qasm/qasm.py

```diff
@@ -854,35 +854,43 @@ def _get_optype_and_params(op: Op) -> Tuple[OpType, Optional[List[float]]]:
     return (optype, params)
 
 
+def hqs_header(header: str) -> bool:
+    return header in ["hqslib1", "hqslib1_dev"]
+
+
 def circuit_to_qasm_io(
     circ: Circuit,
     stream_out: TextIO,
     header: str = "qelib1",
     include_gate_defs: Optional[Set[str]] = None,
 ) -> None:
     """A method to generate a qasm text stream from a tket Circuit"""
-    if (
-        any(
-            circ.n_gates_of_type(typ)
-            for typ in (
-                OpType.RangePredicate,
-                OpType.MultiBit,
-                OpType.ExplicitPredicate,
-                OpType.ExplicitModifier,
-                OpType.SetBits,
-                OpType.CopyBits,
-                OpType.ClassicalExpBox,
-            )
+    if any(
+        circ.n_gates_of_type(typ)
+        for typ in (
+            OpType.RangePredicate,
+            OpType.MultiBit,
+            OpType.ExplicitPredicate,
+            OpType.ExplicitModifier,
+            OpType.SetBits,
+            OpType.CopyBits,
+            OpType.ClassicalExpBox,
         )
-        and header != "hqslib1"
-    ):
+    ) and (not hqs_header(header)):
         raise QASMUnsupportedError(
             "Complex classical gates only supported with hqslib1."
         )
     if include_gate_defs is None:
         include_gate_defs = {"measure", "reset", "barrier"}
         include_gate_defs.update(_load_include_module(header, False, True).keys())
-        stream_out.write('OPENQASM 2.0;\ninclude "{}.inc";\n\n'.format(header))
+
```

Comment:
I asked Dominic to confirm that only one include is necessary. Let's assume so for now (so we can remove this change; if that turns out wrong we can put it back again).

Reply:
Okay, I have removed that for now.